### PR TITLE
Update/10.8.12

### PR DESCRIPTION
**Pull request by k0gen**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,101 +1,67 @@
...
+  <img src="icon.png" alt="Project Logo" width="21%">
+</p>
 
-Jellyfin is a Free Software Media System that puts you in control of managing and streaming your media. It is an alternative to the proprietary Emby and Plex, to provide media from a dedicated server to end-user devices via multiple apps. Jellyfin is descended from Emby's 3.5.2 release and ported to the .NET Core framework to enable full cross-platform support. There are no strings attached, no premium licenses or features, and no hidden agendas: just a team who want to build something better and work together to achieve it. We welcome anyone who is interested in joining us in our quest!
+# Jellyfin for StartOS
+
+[Jellyfin](https://github.com/jellyfin/jellyfin) is a Free Software Media System that puts you in control of managing and streaming your media. It is an alternative to the proprietary Emby and Plex, to provide media from a dedicated server to end-user devices via multiple apps. This repository creates the `s9pk` package that is installed to run `Webtop` on [StartOS](https://github.com/Start9Labs/start-os/). Learn more about service packaging in the [Developer Docs](https://start9.com/latest/developer-docs/).
```
Comment (Dominion5254) [inline README.md:7]: It appears `Webtop` references were copied over mistakenly?

File: README.md [hunk 2]
```diff
@@ -1,101 +1,67 @@
...
-./install-sdk.sh
-embassy-sdk init
-```
-Now you are ready to build the `jellyfin` package!
+- [start-sdk](https://github.com/Start9Labs/start-os/sdk/master/backend)
 
 ## Cloning
 
-Clone the project locally:
+Clone the Webtop package repository locally.
```
Comment (Dominion5254) [inline README.md:22]: Another Webtop reference

File: README.md [hunk 3]
```diff
@@ -1,101 +1,67 @@
...
-cd jellyfin-wrapper
-git submodule update --init --recursive
+git clone git@github.com:Start9Labs/jellyfin-startos.git
+cd jellyfin-startos
 ```
 
 ## Building
 
-To build the `jellyfin` package for all platforms using embassy-sdk version >=0.3.3, run the following command:
+To build the **Webtop** service as a universal package, run the following command:
```
Comment (Dominion5254) [inline README.md:31]: Another Webtop reference

File: README.md [hunk 4]
```diff
@@ -1,101 +1,67 @@
...
 ```
 
-> **Tip:** You can also install the jellyfin.s9pk using **Sideload Service** under the **System > Manage** section.
+**Tip:** You can also install the jellyfin.s9pk by sideloading it under the **StartOS > System > Sideload a Service** section.
 
-### Verify Install
+## Verify Install
 
-Go to your Embassy Services page, select **Jellyfin**, configure and start the service. Then, verify its interfaces are accessible.
+Go to your StartOS Services page, select **Webtop**, configure and start the service.
```
Comment (Dominion5254) [inline README.md:65]: Another Webtop reference

File: icon.png
Comment (Dominion5254) [inline icon.png:1]: The new icon is not centered. Will this impact how the icon appears when running on StartOS?

**Review by Dominion5254 — CHANGES_REQUESTED** [review]
Just a few minor copy changes and this PR looks good.

File: icon.png
Comment (k0gen) [reply icon.png:1]: This is intentional so that it will render correctly in side the circle

**Review by Dominion5254 — APPROVED** [review]

File: instructions.md [hunk 5]
```diff
@@ -4,36 +4,37 @@
 
 1. Config:
 
-* Mediasources - Nextcloud is the default media storage Jellyfin will use as a dependency for media. If you use Filebrowser to store media, you will need to add this as a Media Source. 
+- Mediasources - Nextcloud is the default media storage Jellyfin will use as a dependency for media. If you use Filebrowser to store media, you will need to add this as a Media Source.
```
Comment (kn0wmad) [inline instructions.md]: Media sources

File: instructions.md [hunk 6]
```diff
@@ -4,36 +4,37 @@
...
 
-* Plugins - Jellyfin supports plugins for Chromecast, which requires downloading files from Google servers to support this functionality, as well as YoutubePlayer, which autoloads youtube movie trailers matching media metadata. These plugins have been disabled by default. If you would like to use these features, toggle the corresponding switch to 'on'. Save any Config changes.
+- Plugins - Jellyfin supports plugins for Chromecast, which requires downloading files from Google servers to support this functionality, as well as YoutubePlayer, which autoloads youtube movie trailers matching media metadata. These plugins have been disabled by default. If you would like to use these features, toggle the corresponding switch to **on**. Save any Config changes.
```
Comment (kn0wmad) [inline instructions.md]: "to support this functionality" is redundant
YouTubePlayer / YouTube

File: instructions.md [hunk 7]
```diff
@@ -4,36 +4,37 @@
...
 
-2. Launch the Jellyfin UI, select 'Next', and choose a username and password. We recommend saving this password in a password manager, such as Vaultwarden on your Start9 server. Press 'Next'.
+2. Launch the Jellyfin UI, select **Next**, and choose a username and password. We recommend saving this password in a password manager, such as Vaultwarden on your Start9 server. Press **Next**.
 
 3. Add a Media Library:
 
-* Select 'Add Media Library' followed by the 'Content Type' and 'Display Name' for the library being added. Click the `+` button to the right of 'Folders' and choose the corresponding directory where your media is stored. Jellyfin can support both filebrowser `/mnt/filebrowser` as well as nextcloud `/mnt/nextcloud`.
+- Select **Add Media Library** followed by the **Content Type** and **Display Name** for the library being added. Click the `+` button to the right of **Folders** and choose the corresponding directory where your media is stored. Jellyfin can support both filebrowser `/mnt/filebrowser` as well as nextcloud `/mnt/nextcloud`.
```
Comment (kn0wmad) [inline instructions.md]: This is the most confusing bit, would be awesome to get a community tech to make a 2min video of a jellyfin set up on Nextcloud

**Pull request merged**